README.md

```diff
@@ -68,6 +68,16 @@
 
 You can then check the provisioning status of your cluster by running `oc describe cluster <cluster_name>`
 
+## Developing Cluster Operator Controllers Locally
+
+If you are actively working on controller code you can save some time by compiling and running locally:
+
+  * Run the deploy playbooks normally.
+  * `oc delete DeploymentConfig cluster-operator-controller-manager` - This will be recreated the next time you run the deploy playbooks.
```

Comment:
You should include the namespace parameter `-n openshift-cluster-operator`

You can also just scale it down if you want to bring it back up later as well
`oc scale --replicas=0 dc/cluster-operator-controller-manager -n openshift-cluster-operator`

Reply:
Or if you want other controllers to run except the one you're developing, `oc edit dc/cluster-operator-controller-manager -n openshift-cluster-operator` and add an argument of of --controllers=[comma-separated list of controllers]`

Reply:
Or just `--controllers=-[name of controller you don't want to run]`. For example, if you are working on the infra controller and don't want to run it as part of the deployment, add the argument `--controllers=-infra`.

Reply:
All very handy, the -controller is nice, did not know about that. I will add these all in.